lib/config/types.ts

```diff
@@ -459,6 +458,7 @@ export type RenovateOptions =
 export interface PackageRuleInputConfig extends Record<string, unknown> {
   versioning?: string;
   packageFile?: string;
+  lockFiles?: string[];
```

Comment:
unrelated change?

Reply:
It was missing before, and needed now because of type enforcement of `minimatch()`